Automattic/vip-go-mu-plugins
# FORNO-1632: Remove VaultPress Plugin

## Description

Entirely removes the VaultPress plugin and associated code.

## Changelog Description

### Removed
- Removed the VaultPress plugin and associated code. The plugin has been disabled across the platform since July 2023: https://lobby.vip.wordpress.com/2023/07/11/completed-vaultpress-deprecation/

## Pre-review checklist

Please make sure the items below have been covered before requesting a review:


## Pre-deploy checklist


## Steps to Test

Testing that Jetpack Connection Pilot is unaffected

1. Check out PR.
1. `wp jetpack disconnect blog`
2. `wp jetpack status` 
3. Verify blog is disconnected
4. Run `do_action( 'wpcom_vip_run_jetpack_connection_pilot' );` via `wp shell`
1. `wp jetpack status` 
1. Verify blog is connected
